docs/admin/auth/saml/index.mdx

```diff
@@ -157,6 +157,18 @@ Use the following filters to restrict how users can create accounts and sign in
     }
   ```
 
+**usernameAttributeNames**
+
+  A list of SAML attributes that are used to construct the username for a new user signing up via SAML.
```

Comment:
```suggestion
  A list of SAML attributes to consider for username construction in order or declaration for a new user signing up via SAML.
```

Not sure about my wording here, but somehow wanted to get the ordering in the sentence too :D